dist.sh

```diff
@@ -19,6 +20,9 @@ for i in "$@"; do
   --noredis*)
     $REDIS=0
     ;;
+  --nopagespeed
```

Comment:
change the parameter name to `--skip-pagespeed`. `nopagespeed` is a bit misleading.